Codeherder19/Brians-Travel-Tracker
# Feature/build and test classes

### What does this PR do?
- This PR adds tests and implementation for the Destinations class.
### What does it fix?
- Nothing to fix.

### Is this a feature or a fix?
- This is a feature. Destinations class is set up to take in all the destination data from the respective api, along with a method that will find a destination by id. This method will be used by the Trip class to pass in more data such as estimated costs for lodging and flights, as well as the number of travelers and the duration of the trip.
### Where should the reviewer start?
- Clone this repo. Open this file in your text editor. CD into this directory.
### How should this be tested?
- While in this directory, run npm/test test/destinations-test.js to ensure all tests are passing.
